commands/standalone.js

```diff
@@ -10,10 +17,34 @@ process.env.COZY_FIELDS = JSON.stringify({
 const config = require('../helpers/init-konnector-config')()
 process.env.COZY_URL = config.COZY_URL
 
-const filename = process.argv[2] || process.env.npm_package_main || 'index.js'
+const filename = program.args[0] || process.env.npm_package_main || 'index.js'
```

Comment:
commander.arguments() could be used here ?

Reply:
When I call `yarn standalone` alone, `action` is not called and then it is not a good way to set a default value.

I propose to let it like this and I create a new issue to replace commander by another lib and I propose yargs.